OpenLIFUHome/OpenLIFUHome.py

```diff
@@ -185,7 +186,22 @@ def setParameterNode(self, inputParameterNode: Optional[OpenLIFUHomeParameterNod
             # ui element that needs connection.
             self._parameterNodeGuiTag = self._parameterNode.connectGui(self.ui)
 
+    def onGuidedModeClicked(self):
+        self.logic.toggle_guided_mode_status()
+        slicer.util.selectModule("OpenLIFUData")
```

Comment:
A couple of things with putting `selectModule` here:
- this would switch us to the data module when we click to exit guided mode, which is a bit weird since the data module is really the _start_ of guided mode. clicking "exit" should keep us right where we are in the home module I think.
- the action of "starting guided mode" should be kind of a logic level thing. so for example if from the custom app we want to _start guided mode_ then there should be a way to do that, and we wouldn't do it by calling OpenLIFUHome widet's onGuidedModeClicked; we would probably do it by calling some method of the logic or some method in OpenLIFULib.guided_mode_util.

So I'm not sure what's the best way to design this but what I'm suggesting is that we have some reasonable way for the custom app to issue a python command that for sure turns on guided mode _and_ "starts" it, i.e. puts us into the OpenLIFUData module